ode_test_2.py

```diff
@@ -11,65 +11,73 @@
 from lmfit.model import save_modelresult
 from lmfit.printfuncs import *
 from scipy.integrate import odeint
-import corner 
+import corner
+
+tvec = np.genfromtxt('data_h1n1.csv', usecols=0, delimiter=',',
+                     skip_header=1) #timepoints
+data = np.genfromtxt('data_h1n1.csv', usecols=[1,2], delimiter=',',
+                     skip_header=1) #data
+                     # **usecols must change when changing model states
 
-tvec = np.genfromtxt('data_h1n1.csv', usecols = 0, delimiter = ',', skip_header = 1) #timepoints
-data = np.genfromtxt('data_h1n1.csv', usecols = [1,2], delimiter = ',', skip_header = 1) #data, **usecols must change when changing model states
 
-# 2 state ODE model 
 def f2(u, t, p):
-    try: 
+    """2 state ODE model of Virus and IFN"""
+    try:
         k = p['k'].value
-        K = p['K'].value
-        r_ifn_v = p['r_ifn_v'].value        
+        big_k = p['big_k'].value
+        r_ifn_v = p['r_ifn_v'].value
         d_v = p['d_v'].value
         p_v_ifn = p['p_v_ifn'].value
         d_ifn = p['d_ifn'].value
-    except: 
-        k, K, r_ifn_v, d_v, p_v_ifn, d_ifn = p
-    V, IFN = u 
-    V0 = 6.382
-    IFN0 = 0.26
-    dv = k*V*(1-V/K) - r_ifn_v*(IFN-IFN0)*V - d_v*V
-    difn = p_v_ifn*V - d_ifn*(IFN - IFN0)
+    except:
+        k, big_k, r_ifn_v, d_v, p_v_ifn, d_ifn = p
+    v, ifn = u
+    v0 = 6.382
+    ifn0 = 0.26
+    dv = k*v*(1-v/big_k) - r_ifn_v*(ifn-ifn0)*v - d_v*v
+    difn = p_v_ifn*v - d_ifn*(ifn - ifn0)
     return [dv, difn]
 
-#initial conditions for state, time span 
-u0 = [1.66,0.0]
-tspan = (0., 100.)
 
-# integrator function
 def g(t, u0, p):
+    """Return integration of model"""
     sol = odeint(f2, u0, t, args=(p,))
     return sol
 
 
 def residual(p, t, data):
-    u0 = [6.382, 0.26]
-    model = g(t, u0, p)
+    """Return residual error"""
+    U0 = [6.382, 0.26]
+    model = g(t, U0, p)
```

Comment:
CAPS are for constants. It looks like U0 is defined below in module scope, so if you are changing that here, then it is definitely not a constant. Also, these values [6.382, 0.26] are very specific; are they init values of some sort that we will want to have arguments for?